project.clj

```diff
@@ -50,6 +51,14 @@
                               "phantom-advanced" ["phantomjs"
                                                   "src/test/js/runner.js"
                                                   "src/test/html/advanced.html"]}}
+  
+  ;; Factoring out the duplication of this test selector function causes an error,
```

Comment:
You can do it.  It's a little odd.

You'd do something like
```clj
(defn gen-tests? [t] 
  (some->> t :ns ns-name str (re-matches #"^clara\.generative-tests.*")))
```

Then down here, do
```clj
:default (complement ~gen-tests?)
:generative ~gen-tests?
```